src/MembershipsSDKAdapter.js

```diff
@@ -0,0 +1,64 @@
+import {BehaviorSubject, throwError} from 'rxjs';
+import {DestinationType, MembershipsAdapter} from '@webex/component-adapter-interfaces';
+
+export default class MembershipsSDKAdapter extends MembershipsAdapter {
+  /**
+   * Gets the active members in a meeting
+   *
+   * @static
+   * @param {object} members Members object from meeting, keyed by ID
+   * @returns {Array}
+   * @memberof MembershipsSDKAdapter
+   */
+  static getActiveMembers(members) {
```

Comment:
With Babel we should be able to use the [`private` field proposal for ES6](https://tc39.es/proposal-class-fields/).
```suggestion
  static #getActiveMembers(members) {
```

Reply:
This doesn't work with our eslint config. I'll add a `@private` to the jsdoc.

Reply:
Thinking about this function again, I don't think it should be a method of the class. I think this can work ok being a local module function (that does **_not_** get exported), as:
1. We don't want to expose this as part of the JSON adapter object
2. Only other functions/methods in this module should make use of it